src/fileSystem.js

```diff
@@ -23,7 +23,6 @@ import { stripSymbols, getFileName } from './writeFile';
 const write = (path, data) => new Promise((resolve) => {
   if (!isValid(path)) {
     console.log('path is not valid');
-    return;
```

Comment:
it's not clear for me why you delete this return statement?
is it because it will crash our methods, right?

please give me more information because now it's not clear move for me